### PR TITLE
more dual face cards to ignore from scryfall, added mapping for bab c…

### PR DESCRIPTION
**Pull request by rkonell**
…ards, updated regex for cases with multiple groups of parenthesis

Needed to make more adjustments for another set up cards I scanned.

I'm playing around with an idea to cross-reference the scryfall names by hitting "https://deckbox.org/mtg/{name}" and if that returns a 302 (it redirects to the home page if not found) then defaulting to the single name.  That should remove the need for the "MULTI_NAMES_IGNORE" that I had added. I wanted to see what you thought of that before I spent too much time.

Also I'm curious if you have a linter preference?  I'd like to have my editor lint on save, but I don't want to push up a bunch of formatting changes if they don't match your preferred linting rules.

### CODE REVIEW

File: src/tcg-to-deckbox.py [hunk 1]
```diff
@@ -178,12 +181,14 @@ def getPathPrefix():
...
-
+        # Handle other BaB promo cards
+        if row["Name"] in bab_mapping and row["Edition"] == "Buy-A-Box Promos":
+            row["Edition"] = bab_mapping[row["Name"]]
             
         # For BFZ lands...there's no differentiator from the full arts and the non full arts.
         row["Name"] = row["Name"].replace(" - Full Art", "")
 
-        row["Name"] = re.sub(r" \([^)(]+\)$", "", row["Name"])
+        row["Name"] = re.sub(r" \(.*\)", "", row["Name"])
```
Comment (rkonell) [inline src/tcg-to-deckbox.py:225]: adjusted as I have some cards that tcgplay spits out with multiple groups of parens, e.g. "Blue Sun's Zenith (JP Alternate Art) (Foil Etched)"

**Comment by ThePieBandit** [comment]
Something seems off to me about how it's not registering the card names right for those cards. Let me do some testing. 

As far as linting goes, I think I used something when I first started it, but I don't recall what. I would very much like to have consistent linting, so feel free to update the readme with whatever you want to use.

**Comment by ThePieBandit** [comment]
So, it looks like there's no consistency on which set uses // names and which only list the first face. I'm  going to approve this and we can start to implement your suggestion of checking deckbox to see if the name exists

**Comment by ThePieBandit** [comment]
I pushed a new commit that should resolve the name issue, let me know if you have any issues with it

**Pull request merged**